src/main/java/io/ebeaninternal/server/deploy/parse/AnnotationFields.java

```diff
@@ -282,12 +297,8 @@ private void readField(DeployBeanProperty prop) {
       generatedPropFactory.setUpdateTimestamp(prop);
     }
 
-    if (get(prop, WhoCreated.class) != null) {
-      generatedPropFactory.setWhoCreated(prop);
-    }
-    if (get(prop, WhoModified.class) != null) {
-      generatedPropFactory.setWhoModified(prop);
-    }
+    initWhoCreated(prop);
+    initWhoModified(prop);
 
```

Comment:
What about performance with this extra call ? Not sure about creating a method just for one instruction is a good id here.